bson/src/test/unit/org/bson/codecs/pojo/PojoCustomTest.java

```diff
@@ -545,15 +548,27 @@ public void testInvalidDiscriminatorInNestedModel() {
                         + "'simple': {'_t': 'FakeModel', 'integerField': 42, 'stringField': 'myString'}}"));
     }
 
+    @Test
+    public void testGenericBaseClass() {
```

Comment:
Not sure I needed a new set of classes here, but couldn't find any others that did quite what I wanted to test.